plotly/offline/offline.py

```diff
@@ -298,3 +309,128 @@ def plot(figure_or_data,
             ])
         else:
             return plot_html
+
+
+def plot_mpl(mpl_fig, resize=False, strip_style=False,
+             verbose=False, **kwargs):
+    '''
```

Comment:
:cow2: I think we typically use `"""` not, `'''`. https://www.python.org/dev/peps/pep-0257/